### ISSUE TITLE
[HelpHub]: Link Control Page

### ISSUE THREAD

**Issue by femkreations** [WordPress/Documentation-Issue-Tracker#970]
## New page for 6.3
- [ ] Create a new article for the Link Control page.
- [ ] Make sure the URL slug under Permalink is: link-editing

## Update for 6.3
- [ ] Force disable suggestions until URL field is dirty in Link Control [51354](https://github.com/WordPress/gutenberg/pull/51354)
- [ ] Improve search vs direct URL entry detection in Link Control [51011](https://github.com/WordPress/gutenberg/pull/51011)
- [ ] Visually hide the LinkControl initial/results heading [50977](https://github.com/WordPress/gutenberg/pull/50977)
- [ ] Use "Save" instead of "Apply" in Link Control [50964](https://github.com/WordPress/gutenberg/pull/50964)
- [ ] Use checkbox for Open in new tab within Link Control [50961](https://github.com/WordPress/gutenberg/pull/50961)
- [ ] Reinstate Text control outside of settings in Link Control [50957](https://github.com/WordPress/gutenberg/pull/50957)
- [ ] Link Control require user to manually submit any changes [50668](https://github.com/WordPress/gutenberg/pull/50668)
- [ ] Add support for text only labels [47930](https://github.com/WordPress/gutenberg/pull/47930)
- [ ] Add settings "drawer" to Link Control [47328](https://github.com/WordPress/gutenberg/pull/47328)

## General
- [ ] Make sure all screenshots are relevant to the latest version 
- [ ] Make sure videos, if any, are up to date 
- [ ] Add ALT tags for the images
- [ ] Make sure the headings are in sentence case
- [ ] Convert all reusable blocks to a ‘regular block’.
- [ ] Update the changelog at the end of the article

##WHY?
The way the Link Editing feature works is going to change in [6.3](https://github.com/WordPress/gutenberg/pull/50961) and this is going to affect several articles. It is easier to maintain the updates in one place and link to this page from other articles.

**Comment by zzap** [comment]
Hey @foosantos, thanks for your interest in this issue! 🍪🍪🍪<br>If you have any questions, do not hesitate to ask them in our <a href='https://wordpress.slack.com/archives/C02RP4WU5'>#docs</a> Slack channel.<br>Enjoy and happy contributing ❤️

**Comment by foosantos** [comment]
I had the following tasks:

* Created and published [Link Control](https://wordpress.org/documentation/article/link-editing/)
* Updated [Table Block](https://wordpress.org/documentation/article/table-block/) pointing to the main article
* Updated [Pullquote Block](https://wordpress.org/documentation/article/pullquote-block/) pointing to the main article
* Updated [Preformatted Block](https://wordpress.org/documentation/article/preformatted-block/) pointing to the main article
* Updated [Code Block](https://wordpress.org/documentation/article/code-block/) pointing to the main article
* Updated [Quote block](https://wordpress.org/documentation/article/quote-block/) pointing to the main article
* Updated [List Block](https://wordpress.org/documentation/article/list-block/) pointing to the main article
* Updated [Heading Block](https://wordpress.org/documentation/article/heading-block/) pointing to the main article
* Updated [Paragraph Block](https://wordpress.org/documentation/article/paragraph-block/) pointing to the main article

Could someone review it?

cc @femkreations

**Comment by anandau** [comment]
I have reviewed all the above-mentioned docs. It looks good to me. 

I have one query. Whether such links should open in a new tab? 
As a reader, I am still going through the doc and will click on such links only for some additional info. So I think it can be a good idea to open them in a new tab. That is just my thought. 

What do you think @femkreations @zzap?

**Comment by foosantos** [comment]
@anandau I don't think this is how we generally do on our docs unless I'm missing something here. I guess it could be proposed as a change to the docs.

I also believe it would have accessibility issues — we would need to inform users whenever it opens in a new tab.

**Comment by anandau** [comment]
Yes, it is done in the same way throughout all the docs. Even I have done the same in some of the docs that I have updated recently. It is just a thought that came into my mind while reviewing. 
But I got your point about accessibility.
